### PR TITLE
add FDL order type

### PR DESCRIPTION
**Pull request by frantisekrokusek**
- Add FDL order type
- Add configuration for seamless setup and customization

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,4 +16,5 @@ mkmf.log
 .DS_Store
 sbk.key
 /mt940
-*.key
\ No newline at end of file
+*.key
+.idea/
```
Comment (frantisekrokusek) [inline .gitignore:20]: for RubyMine contributors 🙏

File: lib/epics/client.rb [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 
-  attr_accessor :passphrase, :url, :host_id, :user_id, :partner_id, :keys, :keys_content
+  attr_accessor :passphrase, :url, :host_id, :user_id, :partner_id, :keys, :keys_content, :keys_certs, :locale
   attr_writer :iban, :bic, :name
-  attr_accessor :locale
 
   def_delegators :connection, :post
 
-  def initialize(keys_content, passphrase, url, host_id, user_id, partner_id)
+  def initialize(keys_content, passphrase, url, host_id, user_id, partner_id, locale: Epics.locale)
```
Comment (frantisekrokusek) [inline lib/epics/client.rb:9]: this way you can set it dynamically or from global config

**Comment by tobischo** [comment]
@frantisekrokusek Thank you for the contribution. Always great to see interest in this project.

Given the PR: I think the configuration changes and adding FDL as an order type are 2 separate discussions and should be had separately.

I am willing to get FDL added here.

I am not willing to change how epics is configured and initialized. You described it as `Add configuration for seamless setup and customization`, but for me it adds more complexity and therefore adds seams.

**Comment by frantisekrokusek** [comment]
Closing in favor of https://github.com/railslove/epics/pull/162 and https://github.com/railslove/epics/pull/163

**Pull request closed without merging**